relay/components/Projects/EditProjectMetaForm.tsx

```diff
@@ -64,6 +65,18 @@ const EditProjectMetaForm: React.FC<EditProjectMetaFormProps> = (props: EditProj
                                           transition ease-in-out duration-150" />
                           </div>
                       </div>
+
+                      {!props.hasReleased && <div className="sm:col-span-2">
+                        <label className="block">
+                          <span className="text-gray-700">Project Type</span>
+                          <select value={projectType} onChange={(e) => setProjectType(e.target.value)}
```

Comment:
This line is breaking the build since the projectType state is undefined initially: https://github.com/valist-io/valist/blob/3586ea2e06a3c26e441bd4bd44c5f61d48c41426/relay/components/Projects/EditProjectMetaForm.tsx#L17

We'll need to change it to something like:

```typescript
const [projectType, setProjectType] = useState<ProjectType>('binary');
```

Reply:
actually this would break things -- the default value for this should come from the repo's project type

Reply:
I added:
  const [projectType, setProjectType] = useState<ProjectType>(props.meta.projectType);